plugins/serializers/extr/README.md

```diff
@@ -324,6 +343,50 @@ will serialize into the following extr JSON ouput
       "name": "InterfaceStateChanged",
       "ts": 1654791970
     }
+  ],
+  "ospfNeighborStateChange": [
+    {
+      "device": {
+        "reporterSerialnum": "\"XYZ-5678\"",
+        "serialnum": "\"ABCD-1234\""
+      },
+      "items": [
+        {
+          "keys": {
+            "neighborAddress": "10.20.4.1",
+            "neighborAdressLessInterface": 0,
+            "routerId": 10,
+            "vrf": {
+              "id": 99,
+              "name": "vrf-1"
+            }
+          },
+          "neighborRouterId": 10,
+          "state": {
+            "new": "2Way",
+            "old": "Init"
+          }
+        },
+        {
+          "keys": {
+            "neighborAddress": "10.20.66.1",
+            "neighborAdressLessInterface": 0,
+            "routerId": 10,
+            "vrf": {
```

Comment:
Looking at the NOS-OpenAPI repo,  only vrf_name is used as the key.  vrf_Id is not used as a key.   So we may not need vrf_id in OSPF either.

(NOS-OpenAPI has a key called vr_id, but that is VRRP VR ID, not the usual virtual-router id)

Reply:
Thanks.  Let's consider this as just an example of nested JSON schema syntax, not a real schema. :-)